modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7803,6 +7803,7 @@ from-url=From URL
 from-x=From {0}
 from-x-to-x=From {0} to {1}
 from-your-computer=From Your Computer
+fromName=From Name
```

Comment:
@NemethNorbert the language keys are not camel case, we separate it using -, so it should be from-name instead of fromName, Could you please fix it? Also, if you look into language.properties file we already have this key, so the new key is not needed

Reply:
I made the requested change and force pushed it